strax/context.py

```diff
@@ -1014,6 +965,50 @@ def get_iter(self, run_id: str,
             raise ValueError(f"Invalid time range: {time_range}, "
                              "returned no chunks!")
 
+    def _make_progress_bar(self, run_id, targets, progress_bar=True):
+        """
+        Make a progress bar for get_iter
+        :param run_id, targets: run_id and targets
+        :param progress_bar: Bool weither or not to display the progress bar
+        :return: progress bar, t_start (run) and t_end (run)
+        """
+        try:
+            t_start, t_end, = self.estimate_run_start_and_end(run_id, targets)
+        except (AttributeError, KeyError, IndexError):
+            # During testing some thing remain a secret
+            t_start, t_end, = 0, float('inf')
+        if t_end == float('inf'):
+            progress_bar = False
```

Comment:
I do no think that `self.estimate_run_start_and_end(run_id, targets)` can return `inf` can it? You could remove the if-statement.

Reply:
Yeah so if it cannot find anything, that is where it defaults to:
https://github.com/jorana/strax/blob/eab44a363c2ee83eb5dbe5137a395e5eca2495b2/strax/context.py#L822